src/mon/AuthMonitor.cc

```diff
@@ -1551,8 +1551,12 @@ void AuthMonitor::upgrade_format()
       }
     }
 
-    // add bootstrap key
-    {
+    // add bootstrap key if it does not already exist
+    // (might have already been get-or-create'd by
+    //  ceph-create-keys)
+    EntityName bootstrap_mgr_name;
+    bootstrap_mgr_name.from_str("client.bootstrap-mgr");
+    if (!mon->key_server.contains(bootstrap_mgr_name)) {
       KeyServerData::Incremental auth_inc;
       bool r = auth_inc.name.from_str("client.bootstrap-mgr");
```

Comment:
can't we reuse `bootstrap_mgr_name` here for `auth_inc.name` ?

Reply:
Sure!  done.